casper/src/main/scala/coop/rchain/casper/MultiParentCasperImpl.scala

```diff
@@ -352,26 +337,26 @@ class MultiParentCasperImpl[F[_]
 
     status match {
       case InvalidBlock.AdmissibleEquivocation =>
-        val baseEquivocationBlockSeqNum = block.seqNum - 1
+//        val baseEquivocationBlockSeqNum = block.seqNum - 1
         for {
-          _ <- BlockDagStorage[F].accessEquivocationsTracker { tracker =>
-                for {
-                  equivocations <- tracker.equivocationRecords
-                  _ <- Sync[F].unlessA(equivocations.exists {
-                        case EquivocationRecord(validator, seqNum, _) =>
-                          block.sender == validator && baseEquivocationBlockSeqNum == seqNum
-                        // More than 2 equivocating children from base equivocation block and base block has already been recorded
-                      }) {
-                        val newEquivocationRecord =
-                          EquivocationRecord(
-                            block.sender,
-                            baseEquivocationBlockSeqNum,
-                            Set.empty[BlockHash]
-                          )
-                        tracker.insertEquivocationRecord(newEquivocationRecord)
-                      }
-                } yield ()
-              }
+//          _ <- BlockDagStorage[F].accessEquivocationsTracker { tracker =>
+//                for {
+//                  equivocations <- tracker.equivocationRecords
+//                  _ <- Sync[F].unlessA(equivocations.exists {
+//                        case EquivocationRecord(validator, seqNum, _) =>
+//                          block.sender == validator && baseEquivocationBlockSeqNum == seqNum
+//                        // More than 2 equivocating children from base equivocation block and base block has already been recorded
+//                      }) {
+//                        val newEquivocationRecord =
+//                          EquivocationRecord(
+//                            block.sender,
+//                            baseEquivocationBlockSeqNum,
+//                            Set.empty[BlockHash]
+//                          )
+//                        tracker.insertEquivocationRecord(newEquivocationRecord)
+//                      }
+//                } yield ()
+//              }
```

Comment:
For now, I commented old equivocation code just as a reference for easier tracking where code needs to be updated.

Reply:
Maybe make sense to add `TODO` comment here?